src/classes/BGE_DataImportBatchEntry_CTRL.cls

```diff
@@ -374,6 +407,7 @@ public with sharing class BGE_DataImportBatchEntry_CTRL {
         dataImportModel.totalCountOfRows = getDataImportBatchTotalCount(batchId);
         AggregateResult[] aggResult = getDataImportBatchTotalAmount(batchId);
         dataImportModel.totalRowAmount = (Decimal) aggResult[0].get('amount');
+        dataImportModel.batchTotalRowAmount = getBatchTotalRowAmount(batchId);
```

Comment:
Does this property need to be added onto the model here? It looks like this method might only be used by old BGE.

Reply:
Yes it does, It's called  by `runSingleDryRun` which is we call every time we add a row to a batch in Gift Entry.